docs/checkers/index.md

```diff
@@ -1657,6 +1657,32 @@ class CashRegister:
 - [R0201](#R0201)
 - [StackOverflow: What is the difference between `@staticmethod` and `@classmethod` in Python?]
 
+(W0245)=
+
+### Super without brackets (W0245)
+
+When making a call to a parent class using super(), we must always include the brackets since it is a type of function call. Without the brackets, python may interpret it as an undefined object named "super" rather than actually calling its superclass.
```

Comment:
- Use back ticks (`) around super() to make it appear in code font.
- Capitalize "Python"
- Instead of "as an undefined object..." say "as the `super` function itself rather than calling the function to access the superclass."

- Cap

Reply:
I'm not sure what you mean by the last bullet point "Cap".

Reply:
Ah yes that was just a typo!